src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/DataCollectors/EarningsCollector.cs

```diff
@@ -38,7 +38,8 @@ private static PeriodisedValuesEntity[] ReadEarningsFromDeds()
             {
                 var command = "SELECT "
```

Comment:
not a fan of concatenated strings. Could this be re-written as a verbatim string to make it more readable

Reply:
command could also be a constant

Reply:
Made the command const. but refrained from making it a verbetim string as it doesn't add any value (in this case)